charts/akeyless-api-gateway/templates/_helpers.tpl

```diff
@@ -66,10 +66,10 @@ Create the name of the service account to use
 Get the Ingress TLS secret.
 */}}
 {{- define "akeyless-api-gw.ingressSecretTLSName" -}}
-    {{- if .Values.ingress.existingSecret -}}
-        {{- printf "%s" .Values.ingress.existingSecret -}}
+    {{- if .existingSecret -}}
```

Comment:
need to add a relevant explanation/comment in the values.yaml file for .existingSecret - it can be used for multiple secrets